### PR TITLE
Chores: Validation of @requires tags

### PR DESCRIPTION
**Pull request by TorsteinHonsi**
Chores: Validation of @requires tags. We did not have a unified pattern, and there were pointers to non-existing modules.

### CODE REVIEW

File: ts/Stock/Indicators/EMA/EMAIndicator.ts [hunk 1]
```diff
@@ -66,7 +66,7 @@ class EMAIndicator extends SMAIndicator {
      * @extends      plotOptions.sma
      * @since        6.0.0
      * @product      highstock
-     * @requires     stock/indicators/indicators
+     * @requires     indicators/indicators
```
Comment (pawelfus) [inline ts/Stock/Indicators/EMA/EMAIndicator.ts:69]: Is this the correct path? Shouldn't we include `stock`? So the URL in [API ref ](https://api.highcharts.com/highstock/series.ema) will also be correct? 

For example: https://code.highcharts.com/indicators/indicators.js - 404

The same applies to all other indicators

**Review by pawelfus — COMMENTED** [review]
Just one thing, see below

File: ts/Stock/Indicators/EMA/EMAIndicator.ts (hunk 1 above)
Comment (TorsteinHonsi) [reply ts/Stock/Indicators/EMA/EMAIndicator.ts:69]: Oh, you're right! I didn't think of that.

**Review by pawelfus — APPROVED** [review]
Thanks!

**Pull request merged**